pkg/querytracker/querytracker.go

```diff
@@ -304,6 +325,10 @@ func GetTopPersistentAggs(table string) (map[string]struct{}, map[string]bool) {
 		}
 	}
 
+	irrelevantCols := []string{config.GetTimeStampKey(), "*"}
```

Comment:
nit: Also may be you can create this as a global constant instead of declaring and initializing in the function.

Reply:
When you do that also rename this so that it reflects the intent of this variable better.

Reply:
addressed in the latest commit. We cannot create constant arrays in go. Also creating it as a global var does not seem to call config.GetTimeStampKey(). I think it might be due to the timing of variable and package initialization.